css/style.css

```diff
@@ -93,908 +91,628 @@ h6 {
 }
 
 /*--------------------*/
-
-@font-face {
-  font-family: "icons";
-  src: url("../fonts/icons.ttf?mplvmj") format("truetype"), url("../fonts/icons.woff?mplvmj") format("woff"), url("../fonts/icons.svg?mplvmj#icons") format("svg");
-  font-weight: normal;
-  font-style: normal;
-  font-display: block;
-}
-
-[class^=_icon-]::before,
-[class*=" _icon-"]::before {
-  /* use !important to prevent issues with browser extensions that change fonts */
-  font-family: "icons" !important;
-  font-style: normal;
-  font-weight: normal;
-  font-variant: normal;
-  text-transform: none;
+body {
+  height: 100%;
   line-height: 1;
-  /* Better Font Rendering =========== */
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-._icon-airplane:before {
-  content: "\e903";
-}
-
-._icon-alien:before {
-  content: "\e904";
-}
-
-._icon-aws:before {
-  content: "\e905";
+  font-size: 14px;
+  background: #FAFAFA;
+  color: #252B42;
+  font-weight: 500;
+  font-family: "Montserrat";
 }
 
-._icon-dnk:before {
-  content: "\e908";
+.wrapper {
+  min-height: 100%;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
 }
 
-._icon-facebook:before {
-  content: "\e909";
+._container {
```

Comment:
strange class, .container better

Reply:
made this class technical like a freelancer did. Can't do this on projects?